Source/Run_Methods.sql

```diff
@@ -526,7 +526,11 @@ BEGIN
 END;
 GO
 
+-- @CastToNvarchar is for use with sqlcmd in order to
+-- capture the full output.
+-- e.g. sqlcmd -y0 -Q "EXEC tSQLt.XmlResultFormatter @CastToNvarchar=1"
 CREATE PROCEDURE tSQLt.XmlResultFormatter
+    @CastToNvarchar int = 0

```

Comment:
It should probably be BIT type parameter.

Reply:
True. I've gotten in the habit of using `int` due to working with some DB wrapper APIs that didn't support `bit` properly. Let me change that.